### DISCUSSION TITLE
Drop deprecated setup.py for pyproject.toml

### DISCUSSION THREAD

**Discussion by sodul** [CleanCut/green#267] [Enhancements]
The old setup.py is now officially deprecated in favor of pyproject.toml. Since toml is now part of the standard library as of Python 3.11 I think it would be good to at least add the toml file and eventually remove setup.py. With the latest version of pip we now get warnings about the deprecation.

See https://github.com/pypa/pip/issues/8559

**Comment by CleanCut** [comment]
Adding a `pyproject.toml` file sounds like a good idea. Feel free to open a PR if you like! 😄 

We'll need to leave `setup.py` for now so that older environments don't break.

**Reply by sodul** [reply]
I posted a PR: #268.

Unfortunately it looks like it will need some extra work to play nice with Python 3.5 and 3.6 which are officially deprecated.
